packages/front-end/components/Features/FeatureVariationsInput.tsx

```diff
@@ -277,188 +280,192 @@ export default function FeatureVariationsInput({
+                            >
+                              <a
+                                role="button"
+                                className="ml-2 link-purple small"
+                                onClick={(e) => {
+                                  e.preventDefault();
+                                  setEqualWeights();
+                                }}
+                              >
+                                <PiArrowsClockwise className="mr-1" size={12} />
+                                set equal
+                              </a>
+                            </Tooltip>
+                          )}
+                      </th>
+                    )}
+                  </tr>
+                </thead>
+                <tbody>
+                  {variations && (
+                    <SortableVariationsList
+                      valuesAsIds={idsMatchIndexes}
+                      variations={variations}
+                      setVariations={
+                        !disableVariations ? setVariations : undefined
+                      }
+                    >
+                      {variations.map((variation, i) => (
+                        <SortableFeatureVariationRow
+                          i={i}
+                          key={variation.id}
+                          variation={variation}
+                          variations={variations}
+                          setVariations={
+                            !disableVariations ? setVariations : undefined
+                          }
+                          setWeight={!disableVariations ? setWeight : undefined}
+                          onlySafeToEditVariationMetadata={
+                            onlySafeToEditVariationMetadata
+                          }
+                          customSplit={editingSplits}
+                          valueType={valueType}
+                          valueAsId={valueAsId}
+                          hideVariationIds={hideVariationIds}
+                          hideValueField={hideValueField || !editingIds}
+                          hideSplit={hideSplits}
+                          feature={feature}
+                          showDescription={showDescriptions}
+                          className={sortableClassName}
+                        />
+                      ))}
+                    </SortableVariationsList>
                   )}
+                </tbody>
+              </table>
+              {variations && variations.length > 2 && !isBandit ? (
+                <PremiumCallout
+                  id="exp-implementation-bandit-promo"
+                  commercialFeature="multi-armed-bandits"
+                  docSection="bandits"
+                  dismissable={true}
+                  mt="4"
+                  mb="1"
+                >
+                  Bandits can help you quickly find the best performing variant.
+                </PremiumCallout>
+              ) : null}
```

Comment:
Same here.  Instead of linking to docs for Pro users, we want to just let them convert this to a bandit right away.  This is much trickier to do here vs on the experiment page.  Honestly, might be better to just hide this callout completely for existing Pro users for now.  Just removing the `docSection` prop should do that for you automatically.

Reply:
@jdorn Yea - I think it might get a bit funky to try and convert this to a bandit while the modal is open.

Reply:
Re: `Honestly, might be better to just hide this callout completely for existing Pro users for now. Just removing the docSection prop should do that for you automatically.` 

I'm sorta inclined to leave it in. Since it can be dismissed, I do like the idea of rendering the callout with the link to the docs to give a clear path for someone to learn what bandits are. The user can always dismiss the callout if they'd like.

Thoughts?

Reply:
I'd probably remove it from here.  Let them create the experiment draft without distractions, then they will see the callout on the experiment page where they land on after and can switch to a bandit there.  I think that will be a better UX.